integration-tests/bats/sql-push.bats

```diff
@@ -212,7 +212,7 @@ SQL
     dolt sql -q "call dolt_push('-u', 'origin', 'other')"
     # upstream should be set still
     run dolt push
-    [ "$status" -eq 0 ]
+    [ "$status" -eq 1 ]
```

Comment:
If everything is up to date, exit status 0 is the right thing to do. I verified that this is what git push does.